setup.cfg

```diff
@@ -51,10 +51,12 @@ skip_covered = true
 ignore =
     W503
     W601
+    E203
```

Comment:
Do we need this? I tried removing it and it didn't seem to trigger any warnings.

Reply:
I see this is from https://black.readthedocs.io/en/stable/compatible_configs.html#id1